pymc3/gp/gp.py

```diff
@@ -200,9 +204,9 @@ def conditional(self, name, Xnew, given=None, **kwargs):
         """
         givens = self._get_given_vals(given)
         mu, cov = self._build_conditional(Xnew, *givens)
-        chol = cholesky(stabilize(cov))
+        cov = stabilize(cov)
         shape = infer_shape(Xnew, kwargs.pop("shape", None))
-        return pm.MvNormal(name, mu=mu, chol=chol, shape=shape, **kwargs)
+        return pm.MvNormal(name, mu=mu, cov=cov, shape=shape, **kwargs)
```

Comment:
Does it matter @bwengals?

Reply:
I don't think it does, but why move Cholesky to MvNormal?  i feel it does make the implementation a tiny(!) bit less clear, from the standpoint of someone reading the code.  Is there a benefit?

Reply:
With the graphOp implementation -- which is *not* implemented here after all, but could/might be at some point when and if it proves satisfactory -- the benefit was that this operation happened in the 'subgraph', thus decluttering the total graph, speeding up compilation and freeing up a tiny bit of memory in the process.

From a style point of view, explicitly calling the Cholesky decomposition in the source each time (a good 8 or so times?) might lend the wrong impression that this is the preferred/optimal way to configure a Mv. In truth, Quadbase takes care of the decomposition when given a cov, so either way of configuring it is equally fine. I would find it more coherent to just pass whatever you're working with, as a best practice, since it's shorter and more intuitive.

This is entirely personal though, and has neither computational cost nor benefit. There's also the argument that `stabilize` only makes sense in the context of Cholesky decomposition, and that as it stands here, that context is lost. So I don't mind reverting to the way it was in that regard.

Reply:
Ah ok, it is fine with me either way.  Just wanted to know what the reasoning was.  Thanks!